src/sgame/sg_active.cpp

```diff
@@ -1060,14 +1082,7 @@ static void ClientTimerActions( gentity_t *ent, int msec )
 		     client->pers.aliveSeconds % g_freeFundPeriod.Get() == 0 )
 		{
 			// Give clients some credit periodically
-			if ( client->pers.team == TEAM_ALIENS )
-			{
-				G_AddCreditToClient( client, PLAYER_BASE_VALUE, true );
-			}
-			else if ( client->pers.team == TEAM_HUMANS )
-			{
-				G_AddCreditToClient( client, PLAYER_BASE_VALUE, true );
-			}
+			G_AddCreditToClient( client, PLAYER_BASE_VALUE, true );
```

Comment:
what happens with spectators?
i think they would get credits with this

Reply:
good point, I'll check and update later

Reply:
spectators can not be alive right? But I'd rather keep some kind of check still. This commit is also unrelated to the feature I guess?

Reply:
I've tested this works correctly and added an assertion.